translations/es-ES/events/shallan-meets-wit.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Not sure why github is not showing the diff properly, and it's not easy to spot. The change is in this sentence:
```
Cuando Shallan se interesa por las novedades, **Lin** declara que Helaran queda definitivamente repudiado y nombra a Nan Balat su heredero
```
I'd say this is a typo and it should be Lin, not Lirin (who's Kaladin's father, not Shallan's).